include/freerdp/window.h

```diff
@@ -188,13 +188,13 @@ struct _WINDOW_STATE_ORDER
 	UINT32 ownerWindowId;
 	UINT32 style;
 	UINT32 extendedStyle;
-	UINT32 showState;
+	UINT8 showState;
```

Comment:
While this is the correct value range size type it will break public API, so keep the old one.